firmware/Platypus.cpp

```diff
@@ -309,27 +299,55 @@ void Motor::onLoop_(void *data)
   self->loop();
 }
 
-Sensor::Sensor(int channel) 
-: channel_(channel)
+Sensor::Sensor(int id) : id_(id)
 {  
+  // Abstract Class - should not be instantiated
```

Comment:
I think name() is pure virtual for sensor. The comment is just temporary/to remind people